gamelogs-client/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@tanstack/react-router": "^1.56.5",
+    "gamelogs-client": "file:",
```

Comment:
Out of curiosity, was the lack of this line causing issues? What's the purpose of these additions (here and in the server and root package.json files)?

Reply:
everything added in the package.json files all came from my npm installs. I'm not sure what the changes even were lol.